src/components/Button/BaseButton.tsx

```diff
@@ -159,10 +159,10 @@ export const BaseButton: FC<InternalButtonProps> = React.forwardRef(
         );
 
         return (
-            <>
+            <span className={styles.buttonWrapper}>
```

Comment:
i'm dont think we should wrap every button in a span, is there any other way we can acheive this ?

Reply:
Nope

Reply:
spans have no mass, shouldn't have any effect on layout.